hochschule-der-kunste-bern-musikforschung.csl

```diff
@@ -0,0 +1,492 @@
+              </group>
+            </if>
+            <else>
+              <group delimiter=", ">
+                <text term="ibid"/>
+                <text macro="point-locators-subsequent"/>
+              </group>
+            </else>
+          </choose>
+        </if>
+        <else-if position="ibid">
+          <text term="ibid"/>
+        </else-if>
+        <else>
+          <choose>
+            <if match="all" locator="issue" variable="title-short">
+              <group delimiter=" ">
+                <text variable="title-short" font-style="italic"/>
+                <text variable="locator"/>
+              </group>
+            </if>
+            <else>
+              <choose>
+                <if match="any" variable="author">
+                  <names variable="author">
+                    <name form="short" delimiter="/" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1">
+                      <name-part name="family" text-case="capitalize-first"/>
+                    </name>
+                  </names>
+                </if>
+                <else-if match="any" variable="editor">
+                  <names variable="editor">
+                    <name form="short" delimiter="/" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1"/>
+                  </names>
+                </else-if>
+                <else-if match="any" variable="title-short">
+                  <text variable="title-short"/>
+                </else-if>
+                <else>
+                  <text variable="title"/>
+                </else>
+              </choose>
+              <choose>
+                <if match="any" variable="original-date">
+                  <date date-parts="year" form="text" variable="original-date" prefix=" ">
+                    <date-part name="year"/>
+                  </date>
+                </if>
+                <else>
+                  <date date-parts="year" form="text" variable="issued" prefix=" ">
+                    <date-part name="year"/>
+                  </date>
+                </else>
+              </choose>
+              <text macro="point-locators-subsequent" prefix=", "/>
+            </else>
+          </choose>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0" hanging-indent="true">
+    <sort>
+      <key macro="bibliography-sort"/>
+      <key variable="original-date"/>
+      <key variable="issued"/>
+      <key macro="bibliography-sub-sort"/>
+      <key variable="title"/>
+      <key variable="page"/>
+      <key variable="citation-number"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=" ">
+        <choose>
+          <if match="any" variable="author">
+            <names variable="author" delimiter=", ">
+              <name form="short" delimiter="/" et-al-min="3" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1">
```

Comment:
The subsequent et-al here i not necessary as it's the same rule.
Why not set this globally on line 295?